README.md

```diff
@@ -214,3 +215,37 @@ problem = ABCProblem(bees=10, function=FUNCTIONS['michalewicz'])
 best_bee = problem.solve()
 problem.replay()
 ```
+
+### Grey Wolf Optimization
+
+The Grey Wolf Optimization (GWO) algorithm was proposed by S. Mirjalili et  al. in the paper [Grey Wolf Optimizer](https://www.sciencedirect.com/science/article/pii/S0965997813001853?casa_token=IJMUjrkkBVkAAAAA:TIW5bZqXDqgQwD1aX-5vqsDvmf4AgWFODvWepUiZi79MgSii3MlDMXMwgMVPcQvuGlZYLKm5GA).
+The paper proposed a novel algorithm based on the hunting and the hierarchy structure in grey wolves
+
+#### Features
+
+Enables the GWO algorithm to one of the provided 2D functions. The algorithm tries to find the global minimum of the selected function.
+
+Currently two functions can be selected:
+* [ackley](https://www.sfu.ca/~ssurjano/ackley.html)
+* [michalewicz](https://www.sfu.ca/~ssurjano/michal.html)
+
+![GWO Sample](https://raw.githubusercontent.com/NimishVerma/swarmlib/master/doc/gwo.gif)

```

Comment:
Even it will break the preview for now. Can you please change this URL to point to the target `HaaLeo/swarmlib` repository?